extensions/rich_text_components/base.py

```diff
@@ -49,6 +49,12 @@ class BaseRichTextComponent(object):
     # Whether the component requires the filesystem in some way that
     # prevents it from being used by unauthorized users.
     requires_fs = False
+    # Whether the component should be displayed as a block element
+    is_block_element = False
+    # URL template of the component, such that when interpolated with the
```

Comment:
Returns a URL template which can be interpolated to a URL for the image that represents the component in the RTE. The interpolation dictionary used is the component's customization_args dict, extended with an additional 'explorationId' key whose value corresponds to the id of the containing exploration.